src/commands/api-mesh/create.js

```diff
@@ -31,8 +31,11 @@ class CreateCommand extends Command {
 	static flags = {
 		ignoreCache: ignoreCacheFlag,
 		autoConfirmAction: autoConfirmActionFlag,
+		json: jsonFlag
 	};
 
+	static enableJsonFlag = true
```

Comment:
Can you explain how this fixes the output problem mentioned in the ticket? This value has been set but never used.

Reply:
@revanth0212  sure! Here is the link to oclif docs https://oclif.io/docs/json
>If you want to use the --json flag to return JSON output to the user, then you can set the enableJsonFlag property on the Command class.

>When this property is set and the user supplies the --json flag, the command will suppress all logs and instead log the return value to the console in JSON format. Note log suppression will only work if you use the logging methods on the Command class instance. In other words, this.log will be automatically suppressed but console.log will not be.

For the API Mesh template we need JSON output, since it's easier to parse than a plain text.

Reply:
@revanth0212 it is a static field on the command class so it doesn't necessarily needs to be used in here. That's how it works

Reply:
Ohh so it's an available field on the `Oclif Command` itself 👍 

I was wondering its a field you defined on the Create Command so was checking where was it being used.

This makes sense, thanks 👍